Apps/Examples/Examples/All Examples/DebugMapExample.swift

```diff
@@ -0,0 +1,179 @@
+import UIKit
+import MapboxMaps
+
+private protocol DebugOptionSettingsDelegate: AnyObject {
+    func debugOptionSettingsDidChange(_ controller: SettingsViewController)
+}
+
+private struct MapDebugOptionSetting {
+    let debugOption: MapDebugOptions
+    let displayTitle: String
+}
+
+final class DebugMapExample: UIViewController, ExampleProtocol, DebugOptionSettingsDelegate {
+
+    private var map: MapboxMap!
+
+    override func loadView() {
+        let mapView = MapView(frame: .zero)
+        map = mapView.mapboxMap
+
+        view = mapView
+    }
```

Comment:
These view controllers are configured to allow their view to extend under the navigation bar, which means we can't see the top of the map. You'll either need to change that config on this view controller or move this setup to viewDidLoad and add the map as a subview of the view controller's root view instead of making it the root view itself.